src/glmtools.jl

```diff
@@ -107,8 +130,7 @@ mustart{T<:FP}(d::Distribution,y::Vector{T},wt::Vector{T}) = mustart!(d, similar
 
 type BinomialDevResid <: Functor{3} end
 function evaluate{T<:FP}(::BinomialDevResid,y::T,mu::T,wt::T)
-    omy = one(T)-y
-    2.wt*(xlogy(y,y/mu) + xlogy(omy,omy/(one(T)-mu)))
+    2.wt*ifelse(y == 1, -log(mu), -log1p(-mu))
```

Comment:
A ternary may be faster than `ifelse` here. I'll benchmark (and this shouldn't hold up the merge).